packages/pcd/zk-eddsa-event-ticket-pcd/src/ZKEdDSAEventTicketPCDPackage.ts

```diff
@@ -535,8 +512,12 @@ export async function verify(pcd: ZKEdDSAEventTicketPCD): Promise<boolean> {
   // is available in code as vkey imported above), so doesn't require
   // full package initialization.
 
-  const publicSignals = publicSignalsFromClaim(pcd.claim);
-  return groth16.verify(vkey, publicSignals, pcd.proof);
+  try {
+    const publicSignals = publicSignalsFromClaim(pcd.claim);
+    return groth16.verify(vkey, publicSignals, pcd.proof);
+  } catch (_e) {
```

Comment:
I'm a bit nervous about all the details this is hiding, where bugs in code will look like invalid proofs with no way to find out what's going on.  Not sure if I have a specific suggestion to make it better.  Maybe the error should be logged at least?

Reply:
I suspect it would be fine to drop the try/catch here, and let exceptions bubble up to a caller with a) more context on what's happening and b) access to somewhere to log the error.